exchange/exchange-ps/exchange/New-ApplicationAccessPolicy.md

```diff
@@ -18,6 +18,9 @@ Use the New-ApplicationAccessPolicy cmdlet to restrict or deny access to a speci
 
 For information about the parameter sets in the Syntax section below, see [Exchange cmdlet syntax](https://learn.microsoft.com/powershell/exchange/exchange-cmdlet-syntax).
 
+> [!Note]
+> App Access Policies will soon be replaced by Role Based Access Control for Applications. To learn more, see [Roles Based Access Control for Exchange Applications](https://learn.microsoft.com/exchange/permissions-exo/application-rbac).
```

Comment:
```suggestion
> [!NOTE]
> App Access Policies will soon be replaced by Role Based Access Control for Applications. To learn more, see [Permissions in Exchange Online](/exchange/permissions-exo/permissions-exo#application-rbac).
```

Reply:
Hi @JohanFreelancer9, thanks for the suggestion but the link would have to be 'application-rbac.md'. This is a new article and will be published on 16th Nov, 2022.